Assets/UI/Icons/FontIcons_CQUI.xml

```diff
@@ -3,10 +3,17 @@
     CQUI - fixes for districts' font icons
     Author: Infixo
     2020-08-02: Created
+    2020-08-04: Extended for CityState Banner Suzerain icon (the_m4a)
 -->
 
 <GameInfo>
+    <IconTextureAtlases>
+        <!-- Solid circle with fuzzed edges for blending; used as backing for Suzerain Icon in the City State Banners -->
+        <!-- Note: 44 was the smallest of this style, and did not have a Firaxis-defined IconDefinition  -->
+        <Row Name="ICON_ATLAS_CIRCLE_BACKING_44" IconSize="44" IconsPerRow="1" IconsPerColumn="1" Filename="CircleBacking44.dds"/>
```

Comment:
This file should be used ONLY for fint icons. Firaxis puts all icons into vatious files, but a file that has FontIcons in the name always contans only those. And besides, there is no need to define additional icon.

Reply:
I must be crazy, I looked for a 22.  Didn't see it.  See it now that I directly looked for it.  Weird.  Anyway, thank you!